cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/util/AbstractAppFabricHttpHandler.java

```diff
@@ -298,6 +302,43 @@ protected void getLiveInfo(HttpRequest request, HttpResponder responder, String
     }
   }
 
+  protected void getProgramSchedules(HttpResponder responder, String namespaceId,
```

Comment:
Can this be made a ``private`` method in ``ProgramLifecycleHttpHandler``? Doesn't look like its being used anywhere else.